File: Java-BTL/src/Class/Room.java
/*
 * Click nbfs://nbhost/SystemFileSystem/Templates/Licenses/license-default.txt to change this license
 * Click nbfs://nbhost/SystemFileSystem/Templates/Classes/Class.java to edit this template
 */
package Class;

/**
 *
 * @author 84397
 */
public class Room {
    private String id;
    private boolean isParLab;
    private int numberOfSeat;

    public Room(String id, int numberOfSeat, boolean isParLab) {
        this.id = id;
        this.isParLab = isParLab;
        this.numberOfSeat = numberOfSeat;
    }

    public String getId() {
        return id;
    }

    public void setId(String id) {
        this.id = id;
    }

    public boolean isIsParLab() {
        return isParLab;
    }

    public void setIsParLab(boolean isParLab) {
        this.isParLab = isParLab;
    }

    public int getNumberOfSeat() {
        return numberOfSeat;
    }

    public void setNumberOfSeat(int numberOfSeat) {
        this.numberOfSeat = numberOfSeat;
    }
    
}

File: New/DAL/Room.cs
using System;

public class Room
{
	private string ID;
	private string name;
	private int numberOfSeat;
	bool isParctiacalLab;
	public Room(string ID,string name,int numberOfSeat,bool isParctiacalLab)
	{
		this.isParctiacalLab = isParctiacalLab;
		this.ID = ID;
		this.name = name;
		this.numberOfSeat = numberOfSeat;
	}
	public string getID() { return ID;}
	public string getName() { return name;}
	public int getNumberOfSeat() {  return numberOfSeat;}
	public bool getIsParcticalLab() {  return isParctiacalLab; }

}
